packages/telnyx_webrtc/lib/utils/file_logger.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:io';
```

Comment:
Do the other SDKs have this?

I know we have log levels but do we have a file logger?

If not we should remove this, or at least move it to the client side. 

Right now we are writing to a file for every message and I don't think that's a good idea.